package.json

```diff
@@ -7,6 +7,11 @@
     },
     "extensions": {
       "build": "build"
+    },
+    "devDependencies": {
```

Comment:
this is incorrect. you added it inside devkit section.

you can auto create this config by doing `npm install --save-dev package`